plugins/plugin-factory-github/pom.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
plugins/plugin-factory-github/pom.xml
What if move this to plugin-github
plugins/plugin-github/plugin-github-factory/pom.xml